client/src/app/App.tsx

```diff
@@ -292,6 +295,13 @@ class App extends Component<RouteComponentProps<AppRouteProps> & { config: IConf
                                                     withNetworkProvider(<StardustBlock {...props} />)
                                                 )}
                                             />
+                                            <Route
```

Comment:
Please here add guard to also render if `isStardust` only, else on non-stardust networks we might have collision or routes.